src/main/java/org/mamute/model/Tag.java

```diff
@@ -48,7 +54,8 @@ public Tag() {
 	
 	public Tag(String name, String description, User author) {
 		this.name = name.toLowerCase();
-		this.description = description;
+        this.sluggedName = toSlug(name);
+        this.description = description;
```

Comment:
We use tab for indentation(with size of 4 blank spaces), can you configure your editor for that?